### DISCUSSION TITLE
Why not use DragonFly BSD?

### DISCUSSION THREAD

**Discussion by sirinath** [ravynsoft/ravynos#315] [General]
Looking at the features, at least on paper, [DragonFly BSD](https://www.dragonflybsd.org/) seams the best options to base this on. What at the pros and cons of using the current distribution vs another distribution and why not DragonFly BSD?

**Comment by mszoek** [comment]
What features of DragonFly do you think would make this task easier vs FreeBSD?

**Reply by sirinath** [reply]
I am not thinking it will make it easy. 

But I would like the advertised or claimed performance, scalability, caching and file system features and advantages in DragonFly.

**Reply by lin72h** [reply]
DragonFly' extreme Kernel concurrency is a overkill for desktop use-case(8Core or less). If you ANYTHING other than FreeBSD. All the Linux graphic works is gone, ZFS is gone(Hammer2 maybe a option, but not battle tested like ZFS). Besides the ravynBSD's kernel has Mach IPC and launchd integration, all of them will be gone if we choose other options.

**Reply by lin72h** [reply]
And the DragonFly's lightweight threading model is a mismatch with the pthread_workqueue in kernel.
